### ISSUE TITLE
Enable --all-namespaces option like kubectl

### ISSUE THREAD

**Issue by jjangga0214** [txn2/kubefwd#161]
Hi!

Thanks for creating and maintaining this so convenient tool.

Like `kubectl`, what about providing `--all-namespaces` option?

```shell
kubefwd svc --all-namespaces
```
IMHO, it would be more convenient than `kubefwd svc -n foo -n bar -n baz .... (and more)`.
It'd even exactly match with `kubectl` that provides both of `-n` and `--all-namespaces`. 
It seems so intuitive! 

How do you think?

Thanks.

**Comment by Jimmy-Newtron** [comment]
I was looking for the same feature and I have decided to implement a basic wrapper that does the task:

```shell
#!/bin/bash

# Optional path to kubeconfig
# export KUBECONFIG=~/.kube/config

allNamespaces=$(kubectl get ns --no-headers=true -o custom-columns=Name:metadata.name | paste -sd ' ' - | sed "s/ / -n /g" -)
sudo -E kubefwd svc -n $allNamespaces
```

**Comment by ndj888** [comment]
@cjimti

**Comment by cjimti** [comment]
@ndj888 I can take a look and test tonight or tomorrow. Thanks!

**Comment by cjimti** [comment]
merged in  #204
